pkg/envoy/eds/cluster_load_assignment_test.go

```diff
@@ -65,27 +69,122 @@ func TestNewClusterLoadAssignment(t *testing.T) {
 		},
 		{
 			name:      "no endpoints for cluster",
-			svc:       service.MeshService{Namespace: "ns1", Name: "bookstore-1", TargetPort: 80},
+			svc:       meshSvc,
 			endpoints: nil,
 			expected: &xds_endpoint.ClusterLoadAssignment{
 				ClusterName: "ns1/bookstore-1|80",
 				Endpoints: []*xds_endpoint.LocalityLbEndpoints{
 					{
 						Locality: &xds_core.Locality{
-							Zone: zone,
+							Zone: localZone,
 						},
 						LbEndpoints: []*xds_endpoint.LbEndpoint{},
 					},
 				},
 			},
 		},
+		{
+			name: "multicluster: with remote gateway",
```

Comment:
Please rename this test. This code has no notion of gateway/cluster, just endpoints with weight, priority and zone.